includes/class-wsuwp-people-directory.php

```diff
@@ -415,7 +407,10 @@ public function image_sizes() {
 	 */
 	public function register_meta() {
 		foreach ( $this->post_meta_keys as $key => $args ) {
-			$args['show_in_rest'] = true;
+			if ( 'legacy' === $args['type'] || 'ad' === $args['type'] ) {
+				continue;
+			}
+
 			$args['single'] = true;
 			register_meta( 'post', $key, $args );
```

Comment:
In the switch in how data is organized in `$this->post_meta_keys`, I think this needs to be changed to use the `$args['meta_key']` value instead.